### PR TITLE
Feature/example/redux saga typescript graphql fluentui

### PR DESCRIPTION
**Pull request by bowen31337**
I raise this new PR for addressing previous comments from @lfades

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 12.3s | 12.3s | -21ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.037 | 1.977 | -0.06 |
| / avg req/sec | 1227.51 | 1264.39 | +36.88 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.182 | 1.218 | ⚠️ +0.04 |
| /error-in-render avg req/sec | 2115.49 | 2052.4 | ⚠️ -63.09 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| index.html gzip | 955 B | 955 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 13.4s | 13.7s | ⚠️ +240ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: a131860471cbc3f8fb7ee56afb79d5d95e6a2b4c

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 12s | 12.3s | ⚠️ +316ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.953 | 1.964 | ⚠️ +0.01 |
| / avg req/sec | 1280.17 | 1272.8 | ⚠️ -7.37 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.235 | 1.197 | -0.04 |
| /error-in-render avg req/sec | 2023.8 | 2087.71 | +63.91 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| index.html gzip | 955 B | 955 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 13.7s | 13.9s | ⚠️ +203ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: ecea92827b1ea96582024bb697ef4e14c3c41d69

File: examples/redux-saga-typescript-graphql-fluentui/components/layouts/header.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    backgroundColor: theme.palette.themePrimary,
+    color: theme.palette.white,
+    lineHeight: '50px',
+    padding: '0 20px',
+  },
+}
+
+export const Header: React.FunctionComponent = () => (
+  <Text as="h2" nowrap block styles={renderDetailsRowStyles}>
+    this is a heder
```
Comment (lfades) [inline examples/redux-saga-typescript-graphql-fluentui/components/layouts/header.tsx:17]: The image in the readme also has `heder` and needs the same fix.

```suggestion
    this is a header
```

File: examples/redux-saga-typescript-graphql-fluentui/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# redux-saga-typescript-graphQL-fluentUI example
+
+It is a simple example of how to build next app by using redux, redux-saga, typesript, GraphQL and UI libary FluentUI. you will have to build your own GraphQl endpoint to work with the UI, the UI looks like
```
Comment (lfades) [inline examples/redux-saga-typescript-graphql-fluentui/README.md:3]: > you will have to build your own GraphQl endpoint to work with the UI

The example has to work locally either by just running `npm run dev` (ideally), or provide detailed instructions for how to get it running.

**Review by lfades — CHANGES_REQUESTED** [review]
Check my comments 🕵️

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 11.5s | 11.6s | ⚠️ +28ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.946 | 1.921 | -0.02 |
| / avg req/sec | 1284.8 | 1301.53 | +16.73 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.201 | 1.141 | -0.06 |
| /error-in-render avg req/sec | 2082 | 2190.51 | +108.51 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.8 kB | 5.8 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 948 B | 948 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.6s | -98ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.8 kB | 5.8 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _error.js | 882 kB | 882 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 882 kB | 882 kB | ✓ |
| link.js | 924 kB | 924 kB | ✓ |
| routerDirect.js | 918 kB | 918 kB | ✓ |
| withRouter.js | 918 kB | 918 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: 4a4490153be75ee6aeaae364b196f24f37bcaa6c

**Comment by lfades** [comment]
Closing the PR as stale, if you have the time to update I'll re-open the PR once that's done 👍

**Pull request closed without merging**